dev/com.ibm.ws.transport.http/resources/com/ibm/ws/http/channel/internal/resources/httpchannelmessages.nlsprops

```diff
@@ -185,3 +185,11 @@ trusted.header.origin.invalid.value.useraction=Ensure that configured IP address
 trusted.sensitive.header.origin.invalid.value=CWWKT0039W: The [{0}] configured trustedSensitiveHeaderOrigin value is invalid and cannot be used.
 trusted.sensitive.header.origin.invalid.value.explanation=The specified value must be either a full IP address or correctly formatted hostname.
 trusted.sensitive.header.origin.invalid.value.useraction=Ensure that configured IP addresses contain a value for each segment, and that hostnames do not contain any invalid characters.
+
+config.hsts.invalid.max.age.value=CWWKT0040W: The configured max-age value [{0}] is invalid and cannot be used. The max-age value must be a number equal to or larger than 0.
+config.hsts.invalid.max.age.value.explanation=The specified max-age value must be the number 0 or a larger number.
```

Comment:
larger integer number

Reply:
I've changed the wording to make it 'non-negative integer', thanks having reviewed/recommended a change!